index.html

```diff
@@ -20,54 +20,68 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
+    <div class="menu-bar hidden-menu">
+      <button class="close-btn">
+              <i class="fa fa-times"></i>
+      </button>
+      <div class="menu-mobile">
```

Comment:
- [ ] Kindly make sure your indentation/padding matches the figma design as suggested by the previous reviewer👍🏽 .

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215589525-ccce16da-54d2-4233-9064-77b0d9816c68.png)

FIgma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215589746-5e14f9bd-5128-468d-9823-369dfa6175a3.png)